Components/Bites/src/OgreApplicationContextQt.cpp

```diff
@@ -131,6 +144,21 @@ namespace OgreBites
 
         p.miscParams["externalWindowHandle"] = std::to_string(size_t(window->winId()));
 
+#ifdef OGRE_WAYLAND
```

Comment:
rewrite without #ifdef as with SDL2

Reply:
Done

Reply:
One problematic thing with this cpp-file is the need for including a private Qt header, which is unstable and requires the exact version number of Qt to include it.